templates/search.html

```diff
@@ -1,6 +1,17 @@
 {% extends "base.html" %}
-{% block body %}
+{% block content %}
     <body>
-        <p1>This is the search page.</p1>
+        <form method="GET" action="/search">
+            <div class="form-group">
+              <p><label for="search">Search:</label></p>
+              <input type="text" name = "searchTerm">
+            </div>
+            <br>
+            <button type="submit" class="btn btn-primary">Submit</button>
+          </form>
     </body>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
+
+<form method="GET" action="/search.html">
```

Comment:
I don't think you need anything below line 14, I think that code got committed by mistake. Can you please let me know why you have made a structure for another form over here?

Reply:
I removed the extra form that was there by mistake. Good catch!